### ISSUE TITLE
Add completed callback

### ISSUE THREAD

**Issue by beamercola** [E-Kuerschner/useAudioPlayer#29]
**Describe the solution you'd like**
I'd like a callback when a track has naturally finished playing (not user stopped).

In my case I'm using it to autoplay the next track in playlist.

**Describe alternatives you've considered**
I've tried testing percentage duration, but it resets to zero before hitting `1.0`

**Additional context**
This is how I'm doing it now:

```js
  const { position, duration } = useAudioPosition({ highRefreshRate: true });
  const [current, setCurrent] = useState(tracks[0]);

  useEffect(() => {
    const percent = position / duration || 0;
    if (percent >= 0.99) {
      const currentIndex = tracks.findIndex((t) => t.id === current.id);
      if (currentIndex + 1 < tracks.length) {
        setCurrent(tracks[currentIndex + 1]);
      }
    }
  }, [position, duration]);

```

**Comment by E-Kuerschner** [comment]
@beamercola yeah I agree this lib should provide some way to call back when key howler events have been fired such as the end event

**Comment by E-Kuerschner** [comment]
@beamercola this will be addressed in v0.0.17 which should be released later today. I opted for a boolean flag rather than a callback to keep things consistent with the rest of the hook. You will be able to use the `ended` flag like so:

```javascript
const  { ended } = useAudioPlayer()

useEffect(() => {
  if (ended) {
    // do whatever you need to do
  }
}, [ended])
```

**Comment by beamercola** [comment]
@E-Kuerschner beautiful!  can't wait to implement later!
